pkgs/top-level/aliases.nix

```diff
@@ -1571,7 +1571,7 @@ mapAliases ({
   truecrypt = throw "'truecrypt' has been renamed to/replaced by 'veracrypt'"; # Converted to throw 2022-02-22
   tuijam = throw "tuijam has been removed because Google Play Music was discontinued"; # Added 2021-03-07
   turbo-geth = throw "turbo-geth has been renamed to erigon"; # Added 2021-08-08
-  tvbrowser-bin = throw "tvbrowser-bin was removed because it is now built from sources; use 'tvbrowser' instead"; # Added 2023-02-04
```

Comment:
It should be kept, too. It will not hurt anyone to fix their derivations...

Reply:
I agree in general, however, it seems nicer for stable branch users to give them a grace period where both package names work.  I guess that's one of the main use cases of `aliases.nix`; at least, that's what is being done in similar cases.

Reply:
aliases should not be a throw from the beginning, they will be one in the next release cycle.